mmv1/templates/terraform/examples/cloudfunctions2_private_workerpool.tf.erb

```diff
@@ -0,0 +1,51 @@
+# [START functions_v2_private_workerpool]
```

Comment:
```suggestion
```

Reply:
We generated CGC samples off these for a bit, but have since stopped doing so! We can omit region tags given that.